pymc/tests/test_distributions.py

```diff
@@ -2312,6 +2324,25 @@ def test_stickbreakingweights_invalid(self):
         assert pm.logp(sbw, np.array([0.4, 0.3, 0.2, -0.1])).eval() == -np.inf
         assert pm.logp(sbw_wrong_K, np.array([0.4, 0.3, 0.2, 0.1])).eval() == -np.inf
 
+    @pytest.mark.parametrize(
+        "alpha,K",
+        [
+            (np.array([0.5, 1.0, 2.0]), 3),
+            (np.arange(1, 7, dtype="float64").reshape(2, 3), 5),
+        ],
+    )
+    def test_stickbreakingweights_vectorized(self, alpha, K, stickbreakingweights_logpdf):
```

Comment:
Is `stickbreakingweights_logpdf` passed as an argument here via the fixture sharing decorator of the function?

Reply:
Yes, that's how fixtures work